1.6 Windows: Java error when Serial opened by another instance

Open an IDE, start Serial monitor.
Open another IDE instance, try to upload to this port. Java errors will appear:

```
Sketch uses 7,140 bytes (24%) of program storage space. Maximum is 28,672 bytes.
Global variables use 363 bytes (35%) of dynamic memory, leaving 661 bytes for local variables. Maximum is 1,024 bytes.
Forcing reset using 1200bps open/close on port COM16
processing.app.debug.RunnerException
	at cc.arduino.packages.uploaders.SerialUploader.uploadUsingPreferences(Unknown Source)
	at processing.app.debug.Compiler.upload(Unknown Source)
	at processing.app.Sketch.upload(Sketch.java:1218)
	at processing.app.Sketch.exportApplet(Sketch.java:1191)
	at processing.app.Sketch.exportApplet(Sketch.java:1163)
	at processing.app.Editor$DefaultExportHandler.run(Editor.java:2391)
	at java.lang.Thread.run(Unknown Source)
Caused by: processing.app.SerialException: Error touching serial port 'COM16'.
	at processing.app.Serial.touchPort(Unknown Source)
	... 7 more
Caused by: jssc.SerialPortException: Port name - COM16; Method name - openPort(); Exception type - Port busy.
	at jssc.SerialPort.openPort(SerialPort.java:164)
	... 8 more
```

--- a/app/src/processing/app/Editor.java
+++ b/app/src/processing/app/Editor.java
@@ -26,6 +26,7 @@ import cc.arduino.packages.MonitorFactory;
 
 import com.jcraft.jsch.JSchException;
 
+import jssc.SerialPortException;
 import processing.app.debug.*;
 import processing.app.forms.PasswordAuthorizationDialog;
 import processing.app.helpers.OSUtils;
@@ -2572,6 +2573,12 @@ public class Editor extends JFrame implements RunnerListener {
         statusError(_("Unable to connect: is the sketch using the bridge?"));
       } catch (JSchException e) {
         statusError(_("Unable to connect: wrong password?"));
+      } catch (SerialException e) {
+        String errorMessage = e.getMessage();
+        if (e.getCause() != null && e.getCause() instanceof SerialPortException) {
+          errorMessage += " (" + ((SerialPortException) e.getCause()).getExceptionType() + ")";
+        }
+        statusError(errorMessage);
       } catch (Exception e) {
         statusError(e);
       } finally {

